models/health.go

```diff
@@ -57,21 +61,48 @@ type WorkloadStatus struct {
 
 // RequestHealth holds several stats about recent request errors
 type RequestHealth struct {
-	requestRate float64
-	errorRate   float64
-	ErrorRatio  float64 `json:"errorRatio"`
+	inboundErrorRate    float64
+	outboundErrorRate   float64
+	inboundRequestRate  float64
+	outboundRequestRate float64
+
+	ErrorRatio         float64 `json:"errorRatio"`
```

Comment:
I guess you kept the global error ratio because it is also displayed separately in different lists views?
I don't know if it still makes sense to have error rate displayed both here in in health. Maybe it could be removed in the future. But not for this PR.